components/apply/PositionPreference.tsx

```diff
@@ -31,24 +36,30 @@ const PositionPreference: FC<Props> = ({ values, memberRoles }: Props) => {
   return (
     <div className="grid gap-3 mb-12">
       <h4 className="text-blue-100">Position Preference</h4>
-      <p className="text-charcoal-500 mb-4">
-        Please select the position(s) you are interested in.*
-      </p>
+      {!readOnly && (
+        <p className="text-charcoal-500 mb-4">
+          Please select the position(s) you are interested in.*
+        </p>
+      )}
       <div className="grid gap-6">
         <SelectInput
           id="firstChoiceRole"
           labelText="What is your first choice role?"
           value={values.firstChoiceRole}
           options={memberRoles}
           required
+          readOnly={readOnly}
         />
-        <SelectInput
-          id="secondChoiceRole"
-          labelText="Optional: What is your second choice role? (only select if you want to be considered for this role)"
-          value={values.secondChoiceRole}
-          options={secondChoiceRoles}
-          required={false}
-        />
+        {values.firstChoiceRole != "" && (
```

Comment:
Do you mean secondChoiceRole here?

Reply:
I used the first choice cause the ticket says to hide the second choice if the user has not yet chosen a first choice. So, if the first choice is an empty string (meaning it is in the default option), then the user has not chosen anything yet. But if it is no longer empty, then the user has chosen a first choice - and the second choice select should appear.